### PR TITLE
chore: bump ethers version

### PR DESCRIPTION
**Pull request by bartlomiej94**

### CODE REVIEW

**Review by fionnachan — COMMENTED** [review]
i think we'll have to bump our sdk version and do a release stating it updating ethers from 5.1 to 5.7

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "@ethersproject/address": "^5.0.8",
     "@ethersproject/bignumber": "^5.1.1",
     "@ethersproject/bytes": "^5.0.8",
-    "ethers": "^5.1.0"
+    "ethers": "^5.7.0"
```
Comment (chrstph-dvx) [inline package.json:51]: Can we check what would be the minimal version needed for beamer bridge, and add it as a `peerDependency`?

Comment (bartlomiej94) [reply package.json:51]: Their [frontend repo](https://github.com/beamer-bridge/beamer/blob/main/frontend/package.json#L29) shows version 5.1.1.

What was the context of correlating it to the beamer bridge? Were they the ones having issues with versioning?

**Comment by spsjvc** [comment]
Marked this as draft for now until we discuss the best approach.

**Pull request closed without merging**